functions/transformations/limit_test.go

```diff
@@ -54,6 +78,35 @@ func TestLimit_Process(t *testing.T) {
 				},
 			}},
 		},
+		{
+			name: "with null",
+			spec: &transformations.LimitProcedureSpec{
+				N: 2,
+				Offset: 1,
+			},
+			data: []flux.Table{&executetest.Table{
+				ColMeta: []flux.ColMeta{
+					{Label: "_time", Type: flux.TTime},
+					{Label: "_value", Type: flux.TFloat},
+				},
+				Data: [][]interface{}{
+					{execute.Time(1), nil},
+					{execute.Time(2), nil},
+					{execute.Time(2), 1.0},
+					{execute.Time(2), 1.0},
+				},
+			}},
+			want: []*executetest.Table{{
+				ColMeta: []flux.ColMeta{
+					{Label: "_time", Type: flux.TTime},
+					{Label: "_value", Type: flux.TFloat},
+				},
+				Data: [][]interface{}{
+					{execute.Time(2), nil},
```

Comment:
I think this looks correct. Limit should return nulls.  Filter can remove nulls if they don't want them. resubmit this test in a separate PR after we settle the Builder interface?

Reply:
I can also keep this PR open and rebase/fix once the PR for the builder will be merged. Do you prefer to accept this as it is with the new code for setting nils?

The change in the table builder won't affect this function because it only depends on `AppendColsArrow` that will use the new builder interface.